pyresample/test/test_utils.py

```diff
@@ -522,7 +522,11 @@ def test_get_area_def_from_raster_extracts_proj_id(self):
         crs = CRS(init='epsg:3857')
         source = tmptiff(crs=crs)
         area_def = utils.rasterio.get_area_def_from_raster(source)
-        self.assertEqual(area_def.proj_id, 'WGS 84 / Pseudo-Mercator')
+        epsg3857_names = (
+            'WGS_1984_Web_Mercator_Auxiliary_Sphere',
+            'WGS 84 / Pseudo-Mercator',
```

Comment:
Could you add a small comment saying which version of PROJ each of these names is used in?